lib/pause.js

```diff
@@ -70,23 +76,28 @@ function parseInput(cmd) {
   try {
     const locate = global.locate; // enable locate in this context
     const I = container.support('I');
+    if (cmd.includes('=>')) {
```

Comment:
Nope!
This may match `=>` anywhere. So if someone passes an arrow function inside a command this will detect this...

```suggestion
    if (cmd.trim().startsWith('=>')) {
```